lib/Utility/EncryptionUtil.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:encrypt/encrypt.dart';
+
+
+class EncryptUtil{
+
+ /// Encryption key to use in encrypting and decrypting text notes
+  static final _ENCRYPTION_KEY = 'b4CplJtMrcfEg7LJhL2dZyEwgvHP/75w';
```

Comment:
This should be const not final, final is best used for values not know at runtime 

https://stackoverflow.com/questions/50431055/what-is-the-difference-between-the-const-and-final-keywords-in-dart

but it’s more of a micro compiler optimization I doubt we will notice any difference

Reply:
Same with this one - borrowed. Good article.